File: src/TmrOverlay.App/Overlays/CarRadar/CarRadarOverlayDefinition.cs
using TmrOverlay.Core.Overlays;

namespace TmrOverlay.App.Overlays.CarRadar;

internal static class CarRadarOverlayDefinition
{
    public static OverlayDefinition Definition { get; } = new(
        Id: "car-radar",
        DisplayName: "Car Radar",
        DefaultWidth: 300,
        DefaultHeight: 300,
        Options:
        [
            OverlaySettingsOptionDescriptor.Boolean(
                OverlayOptionKeys.RadarMulticlassWarning,
                "Show multiclass warning",
                defaultValue: true)
        ],
        ShowOpacityControl: false,
        FadeWhenLiveTelemetryUnavailable: true,
        ContextRequirement: OverlayContextRequirement.LocalPlayerInCar);
}

File: local-mac/TmrOverlayMac/Sources/TmrOverlayMac/Overlays/CarRadar/CarRadarOverlayDefinition.swift
import AppKit

enum CarRadarOverlayDefinition {
    static let definition = OverlayDefinition(
        id: "car-radar",
        displayName: "Car Radar",
        defaultSize: NSSize(width: 300, height: 300),
        showOpacityControl: false,
        fadeWhenLiveTelemetryUnavailable: true
    )
}
